alchemy_cms.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |gem|
     activesupport
     railties
   ].each do |rails_gem|
-    gem.add_runtime_dependency rails_gem, [">= 6.0", "< 7.1"]
+    gem.add_runtime_dependency rails_gem, [">= 6.1", "< 7.1"]
```

Comment:
For latest `main` branch I would to drop Rails 6.1 support all together. See #2437